Documentation/ColumnsConfig/Type/Category/Properties/ForeignTableWhere.rst

```diff
@@ -0,0 +1,17 @@
+.. include:: /Includes.rst.txt
+.. _columns-category-properties-foreign-table-where::
+
+=============
+foreign_table_where
+=============
+
+.. confval:: foreign_table_where
+
+   :Path: $GLOBALS['TCA'][$table]['columns'][$field]['config']
+   :type: string (SQL WHERE)
+   :Scope: Proc. / Display
+
+   The items from :ref:`foreign_table <columns-select-properties-foreign-table>`
+   are selected with this WHERE-clause.
```

Comment:
```suggestion
   are selected with this :sql:`WHERE` clause.
```

Reply:
I added your requested changes, added the default value and added a sentence in order to make it clear that the default value should be kept. 
This time I checked and the documentation rendered without warnings for me.